src/flanders/markdown.cljc

```diff
@@ -51,38 +67,48 @@
 (defn- ->leaf-header [this loc]
   (let [type-str (->short-description this)]
     (if (fp/key loc)
-      (->header loc " " type-str " Key")
-      (->header loc " " type-str " Value"))))
-
-(defn- ->reference [{:keys [reference]}]
-  (when (seq reference)
-    (str "* Reference: " reference "\n")))
+      (str "* " type-str " Key\n")
+      (str "* " type-str " Value\n"))))
 
 (defn- ->schema-str [this loc]
-  (str "* Plumatic Schema: "
-       (pr-str (let [schema (fs/->schema this loc)]
-                 (if (fp/sequential? loc)
-                   [schema]
-                   schema)))
-       "\n"))
+  (let [schema (pr-str (fs/->schema this loc))
+        schema (cond
+                 (str/starts-with? schema "(enum") "(enum ...)"
+                 (= "java.lang.String" schema) "Str"
+                 :else schema)]
+    (str "  * Plumatic Schema: "
+         (if (fp/sequential? loc)
+           (str "[" schema "]")
+           schema)
+         "\n")))
 
 (defn- ->values [{v :values}]
   (when (and v (> (count v) 1))
-    (str "* Allowed Values: " (sort (seq v)) "\n")))
+    (str "  * Allowed Values: " (sort (seq v)) "\n")))
```

Comment:
I find this hard to read. Even if it makes more line, I would prefer to have a list of items here.
I don't think the depth is kept (even if it should certainly be 3).

If there are case were it might not be 3 we could use `html` `<ul>` and `<li>` in markdown.

If the depth is 3 that should do it:

~~~clojure
(->> (sort (seq v))
       (map #(str "      * " % "\n"))
~~~